OfficialTutorials/GettingStarted.json

```diff
@@ -48,9 +51,10 @@
       "zh": "《无限飞行》的操纵方式非常直观，只需通过向所需方向倾斜你的设备即可控制飞机的操纵杆。《无限飞行》将检测设备的运动，并将其转换为操纵面的偏转，就好像你自己驾驶飞机一样！",
       "bg": "Нашето приложение има интуитивен начин за управление джойстика или щурвала на самолета - като накланяте устройството си в нужната ви посока. Infinite Flight разпознава това движение и предава сигнала на контролните повърхности, точно както би било, ако управлявате самолета наистина!",
       "ru": "В нашем приложении интуитивно понятный способ управления джойстиком или штурвалом вашего воздушного судна - наклон устройства в нужном направлении. Infinite Flight распознает это движение и передает сигнал на управляющие поверхности так, как если бы вы сами управляли самолетом!",
-      "tr": "Uygulamamız, cihazınızı gitmek istediğiniz yöne doğru eğerek uçağınızın joystick'ini veya boyunduruğunu kontrol etmenin sezgisel bir yolunu sunuyor. Infinite Flight bu hareketi algılar ve sanki uçağı kendiniz uçuruyormuşsunuz gibi kontrol yüzeylerine girdi sağlar! "
+      "tr": "Uygulamamız, cihazınızı gitmek istediğiniz yöne doğru eğerek uçağınızın joystick'ini veya boyunduruğunu kontrol etmenin sezgisel bir yolunu sunuyor. Infinite Flight bu hareketi algılar ve sanki uçağı kendiniz uçuruyormuşsunuz gibi kontrol yüzeylerine girdi sağlar! ",
+      "az": "Tətbiqimiz cihazınızı getmək istədiyiniz istiqamətə əyərək təyyarənizin joystikini və ya boyunduruğunu idarə etmək üçün intuitiv üsul təklif edir. Infinite Flight bu hərəkəti aşkar edir və siz təyyarəni özünüz uçurmuş kimi idarəetmə səthlərinə giriş təmin edir!"
```

Comment:
This translation needs to be rewritten

Reply:
Hello Sorry for the Delayed Reply. If you are okey for it can we continue discussion in Discord?